WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/JetpackScanThreatDetailsViewController.swift

```diff
@@ -0,0 +1,46 @@
+import UIKit
+
+class JetpackScanThreatDetailsViewController: UIViewController {
+
+    // MARK: - Properties
+
+    private let threat: JetpackScanThreat
+
+    // MARK: - Init
+
+    init(threat: JetpackScanThreat) {
+        self.threat = threat
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - View lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        title = Strings.title
+        configureThreatDetailsView()
+    }
+
+    // MARK: - Configure
+
+    private func configureThreatDetailsView() {
+        let threatDetailsView = ThreatDetailsView.loadFromNib()
```

Comment:
🤔 I haven't seen this pattern before so I apologize if my question is weird. 

Is there a benefit to creating a custom UIView (`ThreatDetailsView`) vs just doing the logic inside of the view controller itself?

Reply:
Thought about it, moved the view logic inside the VC 😄
cb10f79